backend/main/test_utils/vote_utils.go

```diff
@@ -26,6 +26,11 @@ func (otu *OverflowTestUtils) GetVotesForProposalAPI(proposalId int) *httptest.R
 	return otu.ExecuteRequest(req)
 }
 
+func (otu *OverflowTestUtils) GetResultsForProposalAPI(proposalId int) *httptest.ResponseRecorder {
+	req, _ := http.NewRequest("GET", "/proposals/"+strconv.Itoa(proposalId)+"/results", nil)
+	return otu.ExecuteRequest(req)
+}
```

Comment:
👍 glad we have a test for this endpoint now

Reply:
actually my bad, we did have a test hitting this endpoint, but not checking for the case of no votes! Just removing the duplicate code now